ui/src/app/kvm/components/KVMResourcesCard/KVMResourcesCard.tsx

```diff
@@ -0,0 +1,75 @@
+import { Card } from "@canonical/react-components";
+import classNames from "classnames";
+import React from "react";
+
+import ContextualMenu from "app/base/components/ContextualMenu";
+import Meter from "app/base/components/Meter";
+
+type Props = {
+  className?: string;
+  title?: string;
+};
+
+const KVMResourcesCard = ({ className, title }: Props): JSX.Element => {
+  return (
+    <Card className={classNames("kvm-resources-card", className)}>
+      {title && (
+        <>
+          <p data-test="kvm-resources-card-title">{title}</p>
```

Comment:
Should this be a `<h5>` or something?

Reply:
Semantically, yeah probably, but the design is just paragraph text. There might be a Vanilla class for default styling actually - I'll use that.